packages/taquito-tzip16/README.md

```diff
@@ -1,51 +1,64 @@
-# Taquito TZip-16 package
+# Taquito TZIP-016 package
+*Documentation can be found [here](https://tezostaquito.io/docs/metadata-tzip16/)*  
+*TypeDoc style documentation is available on-line [here](https://tezostaquito.io/typedoc/modules/_taquito_tzip16.html)*
 
-`@taquito/tzip16` is an npm package that provides developers with Tzip-16 functionality for Taquito. It provides the ability to retrieve metadata associated with a smart contract based on the TZIP-16 standard. The package also provides a way to execute the MichelsonStorageView found in the metadata. More information about the TZIP-16 standard can be found [here](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#introduction).
+`@taquito/tzip16` is an npm package that provides developers with TZIP-016 functionality for Taquito. TZIP-016 is a standard for encoding access to smart contract metadata either on-chain or off-chain. The `@taquito/tzip16` package allows developers to retrieve the metadata associated with a smart contract and execute the off-chain views found in the metadata.
 
-## How to use the tzip16 package
+## General Information
 
-The package can be used as an extension to the well known Taquito contract abstraction. 
+Contract metadata gathers valuable information about the contract that is not directly used for its operation. According to TZIP-016, a contract with metadata must include in its storage a big_map named `%metadata` of the following type: `(big_map %metadata string bytes)`. The big_map must have an empty string as a key where its value is a byte-encoded URI representing the metadata location. Contract metadata can be located on-chain or off-chain (web services or IPFS). Please refer to the following link for complete documentation on [TZIP-016](https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#introduction).
 
-1. **We first need to create an instance of `Tzip16Module` and add it as an extension to our `TezosToolkit`**
+When using the `@taquito/tzip16` package, developers can retrieve metadata of a contract based on its address. The underlying steps performed by Taquito are to find the URI in the contract storage, decode it, extract the metadata accordingly depending on the protocol (tezos-storage, HTTP(S) or IPFS), perform an integrity check if the URI contains a sha256 hash, and returns the metadata to the user.
```

Comment:
returns the metadata to the user.  ->  return the metadata to the user.

can probably drop 'accordingly' from 'metadata accordingly depending '